src/data/set/pairwise.lean

```diff
@@ -3,18 +3,21 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
 -/
-import data.set.lattice
 import logic.relation
+import logic.pairwise
+import data.set.lattice
 
 /-!
 # Relations holding pairwise
 
-This file defines pairwise relations and pairwise disjoint indexed sets.
+This file defines pairwise relations on sets and pairwise disjoint indexed sets.
```

Comment:
https://github.com/leanprover-community/mathlib/pull/17376#discussion_r1014818714

Reply:
@eric-wieser, thanks for copying these across. I will make these suggested fixes in a later PR.

Reply:
Fixed in #17398.